media/js/firefox/private-browsing.js

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+;(function($, Modernizr, site) {
+    'use strict';
+
+    var $html = $(document.documentElement);
```

Comment:
Ahh yes, but actually I've seen $(document.documentElement); on other pages. Wondering why that has been used over $('html')?